packages/nc-gui/composables/useSharedView.ts

```diff
@@ -1,8 +1,18 @@
-import type { ExportTypes, FilterType, KanbanType, PaginatedType, RequestParams, SortType, TableType, ViewType } from 'nocodb-sdk'
+import type {
+  ExportTypes,
+  FilterType,
+  KanbanType,
+  MapType,
+  PaginatedType,
+  RequestParams,
+  SortType,
+  TableType,
+  ViewType,
+} from 'nocodb-sdk'
 import { UITypes } from 'nocodb-sdk'
 import { computed, useGlobal, useMetas, useNuxtApp, useState } from '#imports'
 
-export function useSharedView() {
+export function useSharedView(limit?: number) {
```

Comment:
Adding `limit` as a parameter seems not intuitive as this is used for map view only. Besides, it may get confused with `appInfoDefaultLimit` below. See the below comment.